Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -853,9 +873,8 @@ async def start_listening():
     """
     Starts a Slack SocketMode client and checks for mirrored incidents.
     """
-    await slack_loop()
-    long_loop_task = asyncio.create_task(long_running_loop(), name="Unanswered loop")
-    await asyncio.gather(long_loop_task)
+    tasks = [asyncio.ensure_future(slack_loop()), asyncio.ensure_future(long_running_loop())]
+    await asyncio.gather(*tasks)
```

Comment:
This is that tasks fix. Allows `long_running_loop` to actually run. Could also explain some of the context issues as well.